### PR TITLE
Update to handlebars v5

### PR DESCRIPTION
**Pull request by senekor**
closes https://github.com/rust-lang/www.rust-lang.org/issues/1918

### CODE REVIEW

File: src/i18n.rs [hunk 1]
```diff
@@ -149,55 +149,64 @@ impl TeamHelperParam {
...
         rcx: &mut RenderContext<'reg, 'rc>,
         out: &mut dyn Output,
     ) -> HelperResult {
         let Some(name) = h.param(0) else {
-            return Err(RenderError::new(
+            return Err(RenderErrorReason::ParamNotFoundForIndex(
                 "{{team-text}} must have at least two parameters",
-            ));
+                0,
+            )
```
Comment (senekor) [inline src/i18n.rs:162]: This wasn't a hard error, only a deprecation warning. I hope the variants I chose make some sense. It's a bit late.

File: templates/learn/get-started.html.hbs [hunk 2]
```diff
@@ -137,6 +137,5 @@ fn main() {
            src="/static/images/ferris.gif" />
   </div>
 </section>
-{{> components/tools/install-script }}
```
Comment (senekor) [inline templates/learn/get-started.html.hbs:140]: This file was removed in a commit from 2019. It seems that until now, handlebars silently ignored the missing component. I believe the new error was introduced in [this PR](https://github.com/sunng87/handlebars-rust/pull/571) The result was that the server would crash on the get-started page and the user would see a 404.

Comment (senekor) [reply templates/learn/get-started.html.hbs:140]: I clicked trough a bunch of the pages on the website looking for other occurrances, but couldn't find any. I could've missed something though.

**Review by Manishearth — APPROVED** [review]

**Comment by Manishearth** [comment]
We should carefully check staging before deploying this one. But it should be fine

**Pull request merged**